retropie-shell-script-boilerplate.sh

```diff
@@ -19,8 +19,10 @@ user="$SUDO_USER"
 [[ -z "$user" ]] && user="$(id -un)"
 home="$(eval echo ~$user)"
 
-# If the script is called in '/etc/rc.local' and you're having trouble with the user being 'sudo' instead of 'pi',
-# try commenting the code above and uncommenting the code below.
+# If you really need that the script is run by root user (e.g. script called
```

Comment:
The thing was the opposite. We wanted to use the user `pi` instead of `root` when launching the script in `/etc.rc.local` , that was the purpose of the alternative home method. So I don't how can we rephrase the sentence to say that

Reply:
> We wanted to use the user `pi` instead of `root` when launching the script in `/etc.rc.local`

Yes, we wanted. But we couldn't, remember?

All commands in `/etc/rc.local` are executed at boot **by the root user**.

If we could find a trick to launch scripts at boot by an arbitrary user we wouldn't need the `find` trick at all. The method above would be enough.

Reply:
Got it! You're right! Merging this PR :)